### ISSUE TITLE
More Details Needed for Build tools Documentation

### ISSUE THREAD

**Issue by LarryAzevedo** [twbs/bootstrap#21775]
I've recently been tasked with creating my company's version of Bootstrap which includes building off the source code.  As a designer, I find the [Build tools](https://v4-alpha.getbootstrap.com/getting-started/build-tools/) to be incomplete (because it assumes that you're a developer).

For example, step 2 under [Tooling setup](https://v4-alpha.getbootstrap.com/getting-started/build-tools/#tooling-setup) should probably mention the fact that you have to open a terminal window and where you have to do what in it.  Maybe a video tutorial would help.

Bottom line is... we should try to be more empathetic to others who may not understand this stuff but need to change their own documentation, etc.  BTW, I still haven't been able to build the source code but I'll leave that for another day.

**Comment by KillerJK** [comment]
I'm trying to follow the instructions, but found some problems.
The documentation says "grunt dist" builds the files in /dist (the css and js), but I found out it also runs the "sass-docs" task.

The problem is npm and bower packages (and probably the others) don't include the necessary scss doc files, giving you the error "An output directory must be specified when compiling a directory".

Bootstrap is successfully compiled, but grunt exits with that confusing error. It would be nice if they required files were included, like when downloading the source from github (grunt works fine with that package).

Modifying the grunt file so it doesn't run "sass-docs" would be another option if you don't want to include the whole doc tree. That's what I expected anyway, the documentation seems to indicate "grunt dist" only builds bootstrap, and "grunt docs" only builds the docs.

**Comment by bardiharborow** [comment]
I'm in the process of overhauling the build system, so these issues and more will be fixed soon. <3

**Comment by LarryAzevedo** [comment]
Awesome!  Thank you so much @bardiharborow.  Hopefully it will end up being more intuitive.

**Comment by ricksmt** [comment]
@bardiharborow Congrats on getting [the overhaul done](https://github.com/twbs/bootstrap/commit/b4ac48fdec7637c0c7c69f9d3ff9fc5d21d23676), but could we get the [build tools](https://v4-alpha.getbootstrap.com/getting-started/build-tools/) updated? It was terribly disappointing to finally get everything set up just to find out the instructions were outdated.

**Comment by Johann-S** [comment]
@ricksmt our documentation will be updated in our next release

**Comment by patrickhlauke** [comment]
@ricksmt to expand, https://v4-alpha.getbootstrap.com/ is accurate for v4.0.0-alpha.6

The project repo itself here has progressed beyond that. With the next release (first beta) we'll also have updated build docs. In the meantime you can refer to https://github.com/twbs/bootstrap/blob/v4-dev/docs/4.0/getting-started/build-tools.md

**Comment by mdo** [comment]
Closing since this has been updated in the v4-dev branch.
